src/Dependencies/Collections/Microsoft.CodeAnalysis.Collections.projitems

```diff
@@ -0,0 +1,22 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Consider adding a README.md file here that explains what the purpose of this component is and why developers may want to leverage it.

Reply:
➡️ Added a simple README file